src/coreclr/jit/codegenlinear.cpp

```diff
@@ -1210,13 +1210,6 @@ void CodeGen::genUnspillRegIfNeeded(GenTree* tree)
                 assert(!varTypeIsGC(varDsc));
                 spillType = lclActualType;
             }
-#elif defined(TARGET_ARM64)
-            var_types targetType = unspillTree->gtType;
-            if (spillType != genActualType(varDsc->lvType) && !varTypeIsGC(spillType) && !varDsc->lvNormalizeOnLoad())
```

Comment:
nit: could you also update the comment about to ~genActualType~ GetActualRegisterType()?

Reply:
sure, what would you like to add there? now it looks like:
https://github.com/dotnet/runtime/blob/96ef64d2ed65fb46d394f52d1e50d7a94c6bca71/src/coreclr/jit/lclvars.cpp#L3824-L3830

Reply:
Sorry - didn't notice your comment.

I was suggesting to update the comment right above where you changed the code:
https://github.com/dotnet/runtime/blob/656775f584f8b4dc291d4050d57fbb8466bc15a4/src/coreclr/jit/codegenlinear.cpp#L1195-L1200


~and its size is not the same as genActualType of the type of the lclVar,~
and its size is not the same as actual register type,

Reply:
thanks, fixed